packages-exp/performance-exp/src/utils/errors.ts

```diff
@@ -21,6 +21,8 @@ import { SERVICE, SERVICE_NAME } from '../constants';
 export const enum ErrorCode {
   TRACE_STARTED_BEFORE = 'trace started',
   TRACE_STOPPED_BEFORE = 'trace stopped',
+  NONPOSITIVE_TRACE_START_TIME = '',
```

Comment:
Please add non-empty unique strings as enum values.

Reply:
Thank you Ali! Updated.